conf/llm_factories.json

```
@@ -6,6 +6,12 @@
             "tags": "LLM,TEXT EMBEDDING,SPEECH2TEXT,MODERATION",
             "status": "1",
             "llm": [
+                {
+                    "llm_name": "gpt-4o-mini",
+                    "tags": "LLM,CHAT,128K",
+                    "max_tokens": 128000,
+                    "model_type": "chat,image2text"
```

```suggestion
                    "model_type": "image2text"
```

model_type does not support such value.